core/src/main/java/de/danoeh/antennapod/core/util/playback/PlaybackController.java

```diff
@@ -719,7 +709,8 @@ public float getCurrentPlaybackSpeedMultiplier() {
     }
 
     public boolean canDownmix() {
-        return playbackService != null && playbackService.canDownmix();
+        return (playbackService != null && playbackService.canDownmix())
+                || UserPreferences.useSonic();
```

Comment:
I vaguely recall there was another bug fix with similar pattern: rely on `UserPreferences` when there is no `playbackService`. Any place else we might hit the same type of issue that pop in your mind?

Reply:
I think this only is a problem with the `controller.canXY` methods. Those are now both fixed. There might be more but I don't know of any additional situation like this.